src/game/__tests__/definitions.test.js

```diff
@@ -1,11 +1,18 @@
+import { GAMEMODE_CORNUCOPIA } from '../../utils/constants';
 import { getThreatDescription } from '../definitions';
+import { GAMEMODE_EOP } from '../../utils/constants';
 
 it('produces empty value for non-existent card', () => {
-  const description = getThreatDescription("NN");
+  const description = getThreatDescription("NN", GAMEMODE_EOP);
   expect(description).toBe("");
 });
 
+it('produces empty value for cornucopia gamemode', () => {
+  const description = getThreatDescription("D2", GAMEMODE_CORNUCOPIA);
+  expect(description);
```

Comment:
This doesn't do much, or does it?

Reply:
Yeah, it's probably unnecessary and clogs up the test file. I can remove it.